src/components/form/Label.tsx

```diff
@@ -6,20 +6,20 @@ import { HelpTextContainer } from 'src/components/form/HelpTextContainer';
 import classes from 'src/components/form/Label.module.css';
 import { OptionalIndicator } from 'src/components/form/OptionalIndicator';
 import { RequiredIndicator } from 'src/components/form/RequiredIndicator';
-import { getPlainTextFromNode } from 'src/utils/stringHelper';
 import type { ILabelSettings } from 'src/layout/common.generated';
 
 export interface IFormLabelProps {
-  labelText: React.ReactNode;
+  label: React.ReactNode;
+  labelAsText: string | undefined;
   id: string;
   required?: boolean;
   readOnly?: boolean;
   labelSettings?: ILabelSettings;
   helpText: React.ReactNode;
 }
 
-export function Label(props: IFormLabelProps) {
-  if (!props.labelText) {
+export function Label({ label, labelAsText, id, required, readOnly, labelSettings, helpText }: IFormLabelProps) {
+  if (!label || !labelAsText) {
```

Comment:
Huh, yeah, maybe - I set these to required mostly so that typescript would tell me where they were missing. Anyway, that was before I introduced the `elementAsString()` function that worked with inner `<Lang />` elements, so now it's not even needed at all. Removed it in the latest commit. :raised_hands: